lms/djangoapps/grades/new/course_grade.py

```diff
@@ -112,6 +138,9 @@ def summary(self):
         # doesn't get displayed differently than it gets grades
         grade_summary['percent'] = self.percent
         grade_summary['grade'] = self.letter_grade
+
+        # These two fields are stored separately from percent and grade. The should match unless accessed in between
+        # course and subsection grade updates. If that happens we re-populate them here.
```

Comment:
Eric and I discussed this and we think the race condition it referred to no longer exists. I'm going to delete both comments.